public/js/film.js

```diff
@@ -414,6 +429,53 @@ var availableCerts = [];
             }
         });
     });
+
+
+    // film rating by user 
+
+    $('#modal-film-rate').on('show.bs.modal', function (event) {
+        var id = $(event.relatedTarget).data('id');
+        $.ajax({
+            type: "get",
+            url: "/rating/show/"+ id +"/"+ localStorage.getItem('user_id') +"",
+            success: function(data){
+                console.log(data);
+                var n = $('input:checked').length;
+
+                if(Array.isArray(data) && data.length){
+                    n = data[0].rating_value;
+                    console.log(n);
+                }
+                
+                $("input[name=rate][value=" + n + "]").prop('checked', true);
+            }
+        });
+        $('input:radio').change(function(){
+            var userRating = this.value;
+            console.log(id + " from rate modal");
+            $.ajax({
```

Comment:
Can't handle the correct request and its iterating through database. 
It should work with creating and updating all at once.